src/vellum/workflows/events/types.py

```diff
@@ -57,39 +65,41 @@ class BaseDeploymentParentContext(BaseParentContext):
 
 
 class WorkflowDeploymentParentContext(BaseDeploymentParentContext):
-    type: Literal["WORKFLOW_RELEASE_TAG"] = "WORKFLOW_RELEASE_TAG"
```

Comment:
[blocking] why do we need to remove these defaults? would like to avoid passing the burden onto the consumer of these models

Reply:
At some step while trying to get this working the defaults were an issue, but fixed, and it should be fine now!